db/migrate/20190128142743_create_project_update_priorities.rb

```diff
@@ -0,0 +1,9 @@
+class CreateProjectUpdatePriorities < ActiveRecord::Migration[5.2]
+  def change
+    create_table :project_update_priorities do |t|
+      t.references :project, unique: true
+      t.integer :priority, default: 0
```

Comment:
Should this map to a string instead? Does the integer go off the cardinality of the enum in the class? I think it is a little more work to make it map to a string.